diff --git a/scripts/sparse/eigs.m b/scripts/sparse/eigs.m
index 05735f7..65132ca 100644
--- a/scripts/sparse/eigs.m
+++ b/scripts/sparse/eigs.m
@@ -1403,6 +1403,7 @@ endfunction
 %! opts.maxit = 5;
 %! warning ("off", "Octave:eigs:UnconvergedEigenvalues", "local");
 %! d = eigs (A, 4, "lm", opts);
+%!test <54179>
 %! assert (d(3:4), [NaN+1i*NaN; NaN+1i*NaN]);
 %!testif HAVE_ARPACK
 %! A = magic (100);
